nf-example-topology/README.md

```diff
@@ -0,0 +1,26 @@
+# nf-example-topology
+
+## Description
+An Example Network Function topology for Nephio that enables deployment of workload clusters 
+running the free5gc UPF, SMF, and AMF network functions. Nephio supports deployment of arbitrary combinations
+of these network functions to workload clusters.
+
+## Usage
```

Comment:
I think we should just kill this boilerplate Usage stuff.

We could describe the various resource here and what they are for, and what kind of changes one may want to make in them (can be a follow up PR).